web/packages/design/src/Onboard/OnboardFooter.tsx

```diff
@@ -18,18 +18,15 @@
 
 import React from 'react';
 
-import styled, { useTheme } from 'styled-components';
+import styled from 'styled-components';
 
 import { Flex, Link, Text } from 'design';
 
 export const OnboardFooter = () => {
-  const theme = useTheme();
   return (
     <StyledFooter>
       <StyledContent>
-        <Text typography={theme.typography.paragraph2}>
```

Comment:
Oh, here's a tricky one. If you look at the `getTypography` function from `typography.ts`, you'll notice that it only makes sense if the argument is a string. It can't take literal theme objects, because it simply uses it unconditionally to fetch an item from the `theme.typography` map. Regardless of whether it should work this way, it means that this line's syntax effectively does nothing (indexing the `typography` map with an object yields an `undefined` value). "Fixing" this actually ruins the text (part of the line will be displayed in a different font), and it's been like that ever since the onboard footer component was introduced. So the _current looks_ of it is likely what it was supposed to look like in the first place, and as I'm keeping a rule of not modifying the looks until I'm 100% sure what I'm doing, I removed it as a no-op.

Reply:
That makes total sense, thanks.